Makefile

```
@@ -73,17 +73,19 @@ run: build-local-container kind-load deploy ## Build image and run operator in-c
 
 # Binary builds
 GO_BUILD := $(Q)go build
+GO_OS=$(if $(GOOS),GOOS=$(GOOS))
 build: bin/plain-v0 bin/unpack
 
 bin/plain-v0:
-	CGO_ENABLED=0 go build -o $@ ./provisioner/plain-v0
+	CGO_ENABLED=0 $(GO_OS) go build -o $@ ./provisioner/plain-v0
 
 bin/unpack:
-	CGO_ENABLED=0 go build -o $@ ./cmd/unpack/...
+	CGO_ENABLED=0 $(GO_OS) go build -o $@ ./cmd/unpack/...
 
 build-container: ## Builds provisioner container image locally
 	docker build -f Dockerfile -t $(IMAGE) .
 
+build-local-container: GOOS=linux
```

Ah ha! I missed an `export` in my original suggestion.

I _think_ this is the only change necessary from what exists on `main`.

```suggestion
build-local-container: export GOOS=linux
```

---

Nice, ok that works!